doc/source/development.rst

```diff
@@ -63,6 +64,7 @@ Python modules:
 .. code:: bash
 
     $ zypper in python3-devel libxml2-devel libxslt-devel libffi48-devel glibc-devel gcc
+    $ zypper in texlive-fncychap texlive-wrapfig texlive-capt-of trang
```

Comment:
No errors per se, but two ideas to think about:

1. To reduce installing useless doc packages, add the `--no-recommends` option?
1. Although I'm a big fan of aliases (like `in`), for clarity reasons in documentation I would prefer `install` (would also be consistent with `pip install`).